core/src/main/java/com/oath/oak/KeysManager.java

```diff
@@ -1,60 +0,0 @@
-/**
- * Copyright 2018 Oath Inc.
- * Licensed under the terms of the Apache 2.0 license.
- * Please see LICENSE file in the project root for terms.
- */
-
-package com.oath.oak;
-
-import java.nio.ByteBuffer;
-import java.util.logging.Logger;
-
-class KeysManager<K> {
```

Comment:
So long KeysManager ... 
and thanks for everything

Reply:
I actually still think it should be transferred to EntriesManager: to keep the entries array and all the arithmetic that  comes with it. Probably some time...